### PR TITLE
{vis}[intel/2017a] Mesa v17.0.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2052.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/932828478003f2ab9be767afddab8e1c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2052.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c506a2081663e5f847d7ce02c57707d8 for a full test report.

File: easybuild/easyconfigs/n/nettle/nettle-3.3-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ languages (C++, Python, Pike, ...), in applications like LSH or GNUPG,
+ or even in kernel space."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('GMP', '6.1.2'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/nettle/nettle-3.3-intel-2017a.eb]: this one is missing?

Comment (boegel) [inline easybuild/easyconfigs/n/nettle/nettle-3.3-intel-2017a.eb]: was part of #4260, I've synced this branch with current `develop` now

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 2 out of 3 (6 easyconfigs in this PR)
nic70 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/017614507e7ab1d6dde16167a75e9c4d for a full test report.

**Comment by wpoely86** [comment]
Add this patch to mesa:
```
# Mesa strips out most LLVM compile options but misses `-fp-model source` which causes the build to fail
--- configure.orig	2017-03-20 15:08:58.000000000 +0100
+++ configure	2017-03-31 14:00:50.337329154 +0200
@@ -22317,6 +22317,7 @@
         -e 's/\s\+-O\S*//g' \
         -e 's/\s\+-g\S*//g' \
         -e 's/-fno-rtti\>/-Fno-rtti/g' \
+        -e 's/\s\+-fp-model source//g' \
         -e 's/\s\+-f\S*//g' \
         -e 's/-Fno-rtti\>/-fno-rtti/g' \
         -e 's/^ //'
```
And `V=1` to the buildopts .

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cc6fc8dbe0bd0386afa912e58adfa2bb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2531.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b75069707bbd620d51935694b9e70cf for a full test report.

**Comment by boegel** [comment]
Thanks for the review and the patch @wpoely86!

**Pull request merged**